test/unit/event_store/event_test.rb

```diff
@@ -86,4 +86,42 @@ def test_stale
     expected_stale_events_count = EventStore::Event.where('created_at <= ?', EventStore::Event::TTL.ago).count
     assert_equal expected_stale_events_count, EventStore::Event.stale.count
   end
+
+  class EventRollbacksTest < ActiveSupport::TestCase
+
+    class EventWithCallbacks < EventStore::Event
+      after_rollback :after_rollback_method
+
+      def after_rollback_method; end
+    end
+
+    test 'rolledback harmless exceptions not reported' do
```

Comment:
I'm not following closely your discussion about this, and it took me a while to understand these tests. I suggest to rename them.

```suggestion
    test 'failed rollback without transactional callbacks are not reported' do
```